lib/codecs/MultibaseDecoder.js

```diff
@@ -27,16 +34,20 @@ export class MultibaseDecoder extends CborldDecoder {
     return value;
   }
 
-  static createDecoder({value} = {}) {
+  static createDecoder({value, transformer} = {}) {
+    // use decompression map even if value is not a uint8array
+    const {reverseStringTable} = transformer;
+
     if(!(value instanceof Uint8Array)) {
       return false;
     }
+    const string = reverseStringTable.get(value);
```

Comment:
This looks like it needs to check if `value` is an integer -- and use the reverse string table in that case -- and if it comes back with an `undefined` value, throw an error. You may have just not gotten to this yet.